lib/datahub-client/data_platform_catalogue/client/search.py

```diff
@@ -28,38 +28,28 @@
     SearchResult,
     SortOption,
 )
+from pydantic_core import ValidationError
 
 logger = logging.getLogger(__name__)
 
 
 class SearchClient:
     def __init__(self, graph: DataHubGraph):
         self.graph = graph
-        self.search_query = (
-            files("data_platform_catalogue.client.graphql")
-            .joinpath("search.graphql")
-            .read_text()
-        )
-        self.facets_query = (
-            files("data_platform_catalogue.client.graphql")
-            .joinpath("facets.graphql")
-            .read_text()
-        )
-        self.list_domains_query = (
-            files("data_platform_catalogue.client.graphql")
-            .joinpath("listDomains.graphql")
-            .read_text()
-        )
-        self.get_glossary_terms_query = (
-            files("data_platform_catalogue.client.graphql")
-            .joinpath("getGlossaryTerms.graphql")
-            .read_text()
-        )
-        self.get_tags_query = (
+        self.search_query = self.get_graphql_query("search")
+        self.facets_query = self.get_graphql_query("facets")
+        self.list_domains_query = self.get_graphql_query("listDomains")
+        self.get_glossary_terms_query = self.get_graphql_query("getGlossaryTerms")
+        self.get_tags_query = self.get_graphql_query("getTags")
+
+    @staticmethod
+    def get_graphql_query(graphql_query_file_name: str) -> str:
```

Comment:
🤔 I think this might be simpler as a plain function. Don't think it needs to be part of this class unless you want to treat it as a private method